includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -1412,6 +1403,12 @@ protected function calculate_shipping( $address = [] ) {
 			}
 		}
 
+		// Remove subscription shipping package filter if there is free trial in the cart to allow the calculation of shipping costs.
+		// TODO: revisit this when WC Subscriptions adds support for free trials with shipping.
```

Comment:
Rather than just having a TODO that is super-local to this code, can we create an issue for WooCommerce Subscriptions (or find an existing one), and then link to it from here? It feels like that would be more actionable for whoever looks at this code in future.